src/ocean/Assets.ts

```diff
@@ -638,4 +643,30 @@ export class Assets extends Instantiable {
     }
     return results
   }
+
+  /**
+   *
+   * @param {String} did
+   * @return {Promise<Consumable>}
+   */
+  public async isConsumable(did: string): Promise<Consumable> {
+    const ddo = await this.resolve(did)
+    if (!ddo) return null
+
+    if (ddo.isDisable)
+      return {
+        status: 1,
+        message: 'Asset is disabled'
```

Comment:
we should be more clear what exactly is disabled, and by whom. So maybe like so:

```suggestion
        message: 'Ordering this asset has been temporarily disabled by the publisher.'
```